layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0002-env_default.h-Add-support-for-OS_KERNEL_CMDLINE.patch

```diff
@@ -0,0 +1,77 @@
+Index: git/include/environment/ti/mmc.h
+===================================================================
+--- git.orig/include/environment/ti/mmc.h
++++ git/include/environment/ti/mmc.h
+@@ -14,8 +14,8 @@
+ 	"finduuid=part uuid ${boot} ${bootpart} uuid\0" \
+ 	"args_mmc=run finduuid;setenv bootargs console=${console} " \
+ 		"${optargs} " \
+-		"root=PARTUUID=${uuid} rw " \
+-		"rootfstype=${mmcrootfstype}\0" \
++		"${resin_kernel_root}" \
++		"${os_cmdline}"
+ 	"loadbootscript=load mmc ${mmcdev} ${loadaddr} boot.scr\0" \
+ 	"bootscript=echo Running bootscript from mmc${mmcdev} ...; " \
+ 		"source ${loadaddr}\0" \
+@@ -23,8 +23,8 @@
+ 	"importbootenv=echo Importing environment from mmc${mmcdev} ...; " \
+ 		"env import -t ${loadaddr} ${filesize}\0" \
+ 	"loadbootenv=fatload mmc ${mmcdev} ${loadaddr} ${bootenvfile}\0" \
+-	"loadimage=load ${devtype} ${bootpart} ${loadaddr} ${bootdir}/${bootfile}\0" \
+-	"loadfdt=load ${devtype} ${bootpart} ${fdtaddr} ${bootdir}/${fdtfile}\0" \
++	"loadimage=load ${devtype} ${resin_dev_index}:${resin_root_part} ${loadaddr} ${bootdir}/${bootfile}\0" \
```

Comment:
The changes here are also done in mmc.env. Do we really need to do these changes in both files?

Reply:
Probably not. I first did them on mmc.h and did not work, then switched to mmc.env.

Reply:
Ok, so let's only have the changes where they are getting picked up from.